cockroachdb/docker-compose.cockroachdb.yml

```diff
@@ -17,7 +17,9 @@ services:
             --keystore=v1              
 
     roach1:
-        image: cockroachdb/cockroach:v21.2.3
+        build:
+            context: .
+            dockerfile: cockroach.dockerfile
```

Comment:
let's leave comment here that we need custom docker file to explicitly set proper permissions to the key file due to docker doesn't mount it properly on all systems?